src/wdio/conf.js

```diff
@@ -10,6 +10,11 @@ exports.config = {
   capabilities: [
     {
       browserName: 'chrome',
+      axe: {
+        rules: {
```

Comment:
I would have expected this to be present within the axe config block later in this file

Reply:
I've looked more into this and the axe config block is not passed in with the config parameter in the AxeService.js file.  The axe config here was previously setting inject = 'true', but in the AxeService.js, this was always hardcoded to true and whatever.  The config param matches what is set here in the capabilities; hence, I added the axe config for the rules, I have also now moved the inject config to here and removed the hardcoding from the AxeService.js